.github/workflows/run-keb-chart-install-tests-reusable.yaml

```diff
@@ -0,0 +1,184 @@
+name: Validate KEB chart install (reusable)
+
+on:
+  workflow_call:
+    inputs:
+      last-k3s-versions:
+        description: number of the most recent K3s versions to be used
+        required: false
+        default: 1
+        type: string
+      release:
+        description: 'Determines if the workflow is called from release'
+        default: "true"
+        type: string
+      version:
+        description: 'Release version'
+        default: "0.0.0.0"
+        type: string
+
+jobs:
+  validate-keb-helm-chart:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Get changed files
+      id: changed-files-specific
+      uses: tj-actions/changed-files@90a06d6ba9543371ab4df8eeca0be07ca6054959
+      with:
+        files: |
+          .github/**
+          resources/**
+          scripts/**
+          Makefile
+          **/*.sh
+
+    - name: Prepare chart for next KEB version
+      if: inputs.release == 'true'
```

Comment:
do we have to use comparison in this iff statement?

Reply:
Yes, this is a string.

Reply:
We have agreed to check if boolean works later on.